testcases/oz_web_robot_training/camp3_tis_web.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+
```

Comment:
Same comment as another file. 
1 new line between each session or each test case is enough ka.